scanapi/tree/request_node.py

```diff
@@ -115,6 +115,11 @@ def retry(self):
         return self.spec.get(RETRY_KEY)
 
     def run(self):
+        """Make HTTP requests and generating test results for the given URLs.
+        Returns:
+            [dict]: HTTP response and test results with request node name, 
```

Comment:
```suggestion
            [dict]: HTTP response and test results with request node name,
```

Removing [Trailing whitespace](https://deepsource.io/gh/scanapi/scanapi/run/334be846-b172-4895-807d-b5d64e5e4cb5/python/FLK-W291) 🤦‍♀️